tests/ui/single_component_use_path.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0254]: the name `semver` is defined multiple times
+  --> $DIR/single_component_use_path.rs:6:5
+   |
+LL | extern crate semver;
+   | -------------------- previous import of the extern crate `semver` here
+LL | use semver;
+   |     ^^^^^^ `semver` reimported here
+   |
+   = note: `semver` must be defined only once in the type namespace of this module
```

Comment:
The result is the same with 2015 edtion, even after I add `cx.sess.opts.edition == Edition::Edition2018` branch. I don't know what to do now. I'll try separating `extern crate ident` and `use ident` into different file and see what would happen.

Reply:
I have tried to separate `extern crate ident;` and `use ident` into different file and the lint works in this case. But in order to test this case, I should be able to encorperate a auxiliary file contain an main entry. Or I can drop back to `use std;` which I don't think as a good test, since 'std' is inserted by the compiler, and this might impact the test. I don't know what to do next.

Reply:
this error message is expected. And since we only want to lint the 2018 edition, we shouldn't have any `extern crate` in the test file.

Reply:
Oh, you are right, UI tests didn't include semver as I saw on the root Cargo.toml, I change it to regex, and the test work, perhaps ui tests use other toml.

Reply:
Just use regex then. Shouldn't matter too much.